### DISCUSSION TITLE
Where can I find metadata for pretrained models?

### DISCUSSION THREAD

**Discussion by isaacOnline** [mlfoundations/open_clip#471] [Q&A]
I'm wondering about the best way to find info like the number of samples per epoch or total GMACs for models that show up when you run list_pretrained(). Performance metrics would also be helpful.

I've been trying to search for the info in the repo/on hf but haven't made much progress. E.g. for the RN models trained on cc12m, I haven't been able to find the number of epochs they were trained for, or the total GMACs. 

Really sorry if this is somewhere obvious that I haven't seen. Any guidance would be appreciated!

**Comment by mitchellnw** [comment]
Many of the models have things like `laion2b` or `s34b` in the `--pretrained` string which indicates they are trained for 34b samples seen on the laion2b dataset.
